nodejs/week2/meal-sharing/src/backend/api/meals-router.js

```diff
@@ -0,0 +1,68 @@
+const { query } = require("express");
+const express = require("express");
+const router = express.Router();
+
+const meals = require("./../data/meals.json");
+
+router.get("/", async (request, response) => {
+  try {
+    const supportedParams = ["maxPrice", "limit", "title", "createdAfter"]; //will be needed for query validation
+    const maxPrice = parseInt(request.query.maxPrice); //converting to int
+    const limit = parseInt(request.query.limit); //converting to int
```

Comment:
What would happen if price or limit parameters are not integers?

Reply:
will do it